src/I18n/DateFormatTrait.php

```diff
@@ -323,20 +329,20 @@ public static function parseDateTime(string $time, $format = null)
                 is_subclass_of(static::class, MutableDate::class);
         }
 
-        $defaultTimezone = static::$_isDateInstance ? 'UTC' : date_default_timezone_get();
```

Comment:
looks you killed the `UTC` here

Reply:
That is on purpose to match how Chronos Date works.